scanner/templates/laravel/Dockerfile

```diff
@@ -1,4 +1,4 @@
-# syntax = docker/dockerfile:experimental
+# syntax = .fly/dockerfile:experimental
```

Comment:
This should not be changed to `.fly` right? Seems like a find-and-replace error

Reply:
Yep, thanks!